css/main.css

```diff
@@ -0,0 +1,954 @@
+    padding-right: 60px;
+    list-style: none;
+    font-family: Roboto;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 1.2rem;
+    color: #01010D;
+}
+
+.button-one {
+    background: #1617DB;
+    border-radius: 6px;
+    font-size: 1rem;
+    color: white;
+    font-family: Poppins;
+    font-style: normal;
+    font-weight: 600;
+    padding: 15px 30px;
+    border: none;
+}
+
+.text-one-block {
+    display: flex;
+    width: 100%;
+    flex-grow: 1;
+    align-items: center;
+}
+
+.span-h-p-one-block {
+    padding-left: 100px;
+    width: 46%;
+}
+
+.your-watch {
+    letter-spacing: 0.05em;
+    text-transform: uppercase;
+    color: #1617DB;
+    font-family: Poppins;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+}
+
+.have-you {
+    font-family: Poppins;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 4rem;
+    width: 71%;
+    line-height: 124%;
+    margin: 0;
+    color: #01010D;
+}
+
+.the-styles {
+    width: 79%;
+    font-family: Poppins;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 1rem;
+    line-height: 150%;
+    color: #01010D;
+}
+
+.get-button {
+    background: #1617DB;
+    border-radius: 6px;
+    font-size: 1rem;
+    color: white;
+    font-family: Poppins;
+    font-style: normal;
+    font-weight: 600;
+    padding: 15px 30px;
+    border: none;
+}
+
+.gallery-button {
+    font-family: Poppins;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 1rem;
+    text-transform: uppercase;
+    border: 1px solid rgba(22, 23, 219, 0.2);
+    border-radius: 6px;
+    color: #6565EC;
+    text-align: center;
+    background: rgba(255, 255, 255, -1.5);
+    padding: 14px 30px;
+    margin-left: 30px;
+}
+
+.gallery-button img {
+    position: relative;
+    top: 3px;
+    padding-right: 14px;
+}
+
+.two-block-text {
+    text-align: center;
+    padding-top: 80px;
+}
+
+.h2-two-block {
+    font-family: Poppins;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 3.1rem;
+    line-height: 120%;
+    text-transform: capitalize;
+    color: #01010D;
+    margin: 0;
+}
+
+.p-block-two {
+    font-family: Poppins;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 1rem;
+    line-height: 160%;
+    width: 39%;
+    color: #01010D;
+    margin: 0 auto;
+    padding-top: 25px;
+}
+
+.watch-description {
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+}
+
+.one-description {
+    width: 30%;
+}
+
+.img-description {
+    width: 30%;
+    display: flex;
+    align-items: center;
+}
+
+.two-description {
+    width: 30%;
+}
+
+.android {
+    padding-bottom: 70px;
+    text-align: center;
+}
+
+.android img {
+    padding-bottom: 30px;
+}
+
+.internet {
+    position: relative;
+    bottom: 10px;
+    text-transform: capitalize;
+    font-family: Poppins;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 1.2rem;
+    line-height: 31px;
```

Comment:
Whenyou set value for line-=height - never use PX, always it should be relative. in ideal **em**